.ruff.toml

```diff
@@ -0,0 +1,89 @@
+line-length = 100
+
+lint.select = [
+  "E", # pycodestyle, errors
+  "F", # Pyflakes
+  "I", # isort
+  "W", # pycodestyle, warnings
+]
+
+lint.extend-select = [
```

Comment:
I'd prefer to use mostly stock options, such as in the config I shared with you elsewhere. happy to merge as-is and tweak later, or you you change it now; up to you.